app/ui/browser/views/navbar/location.jsx

```diff
@@ -111,7 +110,10 @@ export class Location extends Component {
   }
 
   componentDidMount() {
-    this.props.ipcRenderer.on('focus-urlbar', () => this.refs.input.select());
+    this.props.ipcRenderer.on('focus-url-bar', () => {
```

Comment:
Ah, already pushed this in https://github.com/mozilla/tofino/commit/1c24497ea6a1fbf08d1cf710635ab54a74230f63 and https://github.com/mozilla/tofino/commit/e0a064eaebeb2ce252e756e37997408b3037a433.  Let's move that into a follow up